packages/api/package.json

```diff
@@ -53,7 +55,7 @@
     "jsonwebtoken": "^8.5.1",
     "mandrill-api": "^1.0.45",
     "node-cache": "^5.1.2",
-    "node-fetch": "^2.6.0",
+    "node-fetch": "^3.2.4",
```

Comment:
node-fetch 3.x fails tests in jest, probably due to node version. Can we maybe keep this to the same version as before?

Reply:
yesterday I merged master and did a little bit of more fixing/cleanup but didn't want to spend much more time on that